src/DynamoMLDataPipeline/DynamoMLDataPipeline.cs

```diff
@@ -219,7 +220,7 @@ static void DataExchangeToForge(string filePath, RestClient client, string token
 
             uploadBinaryRequest.AddHeader("Content-Type", ContentType.Binary);
 
-            uploadBinaryRequest.AddOrUpdateParameter(ContentType.Binary.ToString(), base64CompressedBuffer);
+            uploadBinaryRequest.AddOrUpdateParameter(ContentType.Binary.ToString(), base64CompressedBuffer, ParameterType.RequestBody);
```

Comment:
@reddyashish I have been testing the pipeline on the current status of the branch - if we switch the content type to `text/txt` in this line, we solve the problem with getting the unwanted headers in the response body when we retrieve back the binary. After that the conversion to .dyn works as expected:
`uploadBinaryRequest.AddOrUpdateParameter("text/txt", base64CompressedBuffer, ParameterType.RequestBody)`

Reply:
sounds good, changed it.